### PR TITLE
Edge SIDC - Add option to control edgeEnabled flag

### PR DESCRIPTION
**Pull request by zafrirron**
`edgeEnabled` Adds small sidc icons around edge of map for things with SIDC markers just outside of view.
This option was set to true by default with no way to remove it, added enable disable command.

### CODE REVIEW

File: worldmap/worldmap.js [hunk 1]
```diff
@@ -3066,6 +3066,9 @@ function doCommand(cmd) {
         document.getElementById("maxage").value = cmd.maxage;
         setMaxAge();
     }
+    if (cmd.hasOwnProperty("edgeenabled")) {
+        edgeEnabled = cmd.edgeenabled;
```
Comment (dceejay) [inline worldmap/worldmap.js:3070]: Do we need to check it's a boolean ? or force it to be...

**Comment by zafrirron** [comment]
Sorry ,I’m on a business trip for the last two weeks, hope I’ll get to it this weekend, it very simple fix so I guess by Sunday it will be fixed and tested, if you can wait, until then. Zafi Ron - IphoneOn 25 Sep 2024, at 0:19, Dave Conway-Jones ***@***.***> wrote:﻿
Hi @zafrirron - any chance to resolve so we can merge  ?
Thanks

**Comment by dceejay** [comment]
Sure no rush. Thanks

**Comment by zafrirron** [comment]
fixing

**Pull request closed without merging**